tests/torch/test_compile_keras.py

```diff
@@ -58,19 +25,11 @@ def compile_and_test_keras(
     input_shape = (input_output_feature,)
     n_examples = 50
 
-    # Define the Keras model
-    keras_model = model()
-    keras_model.build((None,) + input_shape)
-    keras_model.compute_output_shape(input_shape=(None, input_output_feature))
-
     # Create random input
     inputset = numpy.random.uniform(-100, 100, size=(n_examples, *input_shape))
 
     # Convert to ONNX
-    output_onnx_file_path = Path(tempfile.mkstemp(suffix=".onnx")[1])
-    onnx_model, _ = tf2onnx.convert.from_keras(
-        keras_model, opset=opset, output_path=str(output_onnx_file_path)
-    )
+    onnx_model = onnx.load(f"tests/data/tf_onnx/fc_{input_output_feature}.onnx")
```

Comment:
not sure where this file is coming from, it does not appear in this PR !

Reply:
I added the missing files

Reply:
thanks